docs: describe how to manage embedded plugin registry for VS Code editor (#2503)
codenvy/che-docs: modules/administration-guide/pages/managing-ide-extensions.adoc

[id="managing-ide-extensions"]
= Managing IDE extensions

> All IDEs use extensions or plugins to extend the functionality. The mechanism of adding extensions into IDEs is different from one to another. 

Reviewer: ```suggestion
IDEs use extensions or plugins to extend their functionality; however, the mechanism for adding extensions differs between editors. 
```
